layers/meta-resin-edison/recipes-core/images/resin-image-initramfs.bbappend

```diff
@@ -0,0 +1 @@
+IMAGE_ROOTFS_MAXSIZE = "16384"
```

Comment:
If the boot partition has less free space than the biggest file (actually the biggest file that we update to), we will block or make our life harder when we will integrate resinhup with this board. That's what I meant by saying it breaks resinhup. It breaks the ability to add support for this board in the future.